.gitignore

```diff
@@ -21,3 +21,4 @@ cabal.sandbox.config
 # extra
 tmp
 .stack-work
+stack.yaml.lock
```

Comment:
How come you wanted to remove the stack.yaml.lock? I thought it was generally recommended to check that into source control https://docs.haskellstack.org/en/stable/lock_files/#lock-files